source/includes/usage-examples/code-snippets/insertMany.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+
+	var uri string
+	if uri = os.Getenv("MONGODB_URI"); uri == "" {
+		log.Fatal("You must set your `MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
+	}
```

Comment:
Should this declare the `uri` as a const above `main`?

There is some inconsistency between the usage examples.
1. insertOne.go follows this pattern of checking from the environment.
2. [deleteOne.go](https://github.com/mongodb/docs-golang/blob/a595baaf3bcf5bac9bdd7c5811edb4fefdaf4bce/source/includes/usage-examples/code-snippets/deleteOne.go#L13-L14) declares the `uri` as a const.

The style reference document notes (2). I added a note linking to deleteOne.go as an example.

Reply:
We're going to update all of the usage examples to include this uri declaration format. The reason is this makes it much easier to run them and include them without modifying any code. The style doc states to get them from the environment.